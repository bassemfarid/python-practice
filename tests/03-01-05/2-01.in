245.2
178.5
13949.3
193.2
19384.6
3922.1
49697.4
191910.134
10394.2
19305.32