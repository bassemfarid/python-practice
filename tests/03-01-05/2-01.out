30917.9954